docs/modules/user-guide/partials/proc_adding-commands-to-a-devfile.adoc

```diff
@@ -54,59 +54,22 @@ components:
       mountSources: true
       command: ['sleep', 'infinity']
       env:
-        - name: GOPATH
+        - name: gopath
```

Comment:
Can you proofcheck golang doesn't expect this environment variable to be really `GOPATH`?

Reply:
@maysunfaisal, the capitalization has been a concern for `gopath` and `goache` . Is it cool if we keep it capitalized?

Reply:
yes, pls do `GOPATH` & `GOCACHE`

Reply:
done!